code/modules/research/designs/weapon_designs.dm

```diff
@@ -134,26 +93,6 @@
 	build_path = /obj/item/grenade/chem_grenade/adv_release
 	category = list("Weapons")
 
-/datum/design/xray
```

Comment:
Might wannanleave xray because bleb

Reply:
Read the comments before posting.

Reply:
Ah. Missed that part.
Still might need to be accompanied by blob nerfs in a followup pr.